code/clue.js

```diff
@@ -1,59 +1,265 @@
-// STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
-
 const mrGreen = {
-  firstName: 'Jacob',
-  lastName: 'Green',
-  color: 'green',
-  description: 'He has a lot of connections',
+  firstName: "Jacob",
+  lastName: "Green",
+  color: "green",
+  description: "He has a lot of connections",
   age: 45,
-  image: 'assets/green.png',
-  occupation: 'Entrepreneur'
-}
+  image: "assets/green.png",
+  occupation: "Entrepreneur"
+};
+
+const professorPlum = {
+  firstName: "Victor",
+  lastName: "Plum",
+  color: "purple",
+  description: "Values knowledge over looks",
+  age: 64,
+  image: "assets/plum.png",
+  occupation: "Professor"
+};
+
+const missScarlet = {
+  firstName: "Cassandra",
+  lastName: "Scarlet",
+  color: "red",
+  description: "Has lips as red as roses",
+  age: 32,
+  image: "assets/scarlet.png",
+  occupation: "Florist"
+};
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+const mrsPeacock = {
+  firstName: "Eleanor",
+  lastName: "Peacock",
+  color: "blue",
+  description: "She brings her binoculars with her everywhere",
+  age: 40,
+  image: "assets/peacock.png",
+  occupation: "Ornithologist"
+};
+
+const colonelMustard = {
+  firstName: "Jack",
+  lastName: "Mustard",
+  color: "yellow",
+  description: "Goes to a barber every week to trim his beard",
+  age: 71,
+  image: "assets/mustard.png",
+  occupation: "Condiment expert"
+};
+
+const mrsWhite = {
+  firstName: "Skyler",
+  lastName: "White",
+  color: "white",
+  description: "Has at least one coffee stain on her white blouse at all times",
+  age: 45,
+  image: "assets/white.png",
+  occupation: "Accountant"
+};
 
 const rope = {
-  name: 'rope',
+  name: "rope",
   weight: 10
-}
+};
+
+const knife = {
+  name: "knife",
+  weight: 5
+};
+
+const candlestick = {
+  name: "candlestick",
+  weight: 15
+};
+
+const poison = {
+  name: "poison",
+  weight: 2
+};
+
+const dumbbell = {
+  name: "dumbell",
+  weight: 25
+};
+
+const axe = {
+  name: "axe",
+  weight: 30
+};
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const bat = {
+  name: "bat",
+  weight: 18
+};
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const trophy = {
+  name: "trophy",
+  weight: 22
+};
+
+const pistol = {
+  name: "pistol",
+  weight: 6
+};
 
 const suspects = [
   mrGreen,
+  professorPlum,
+  missScarlet,
+  mrsPeacock,
+  colonelMustard,
   mrsWhite
-  // ...  and the rest
-]
+];
+
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+];
+
+const rooms = [
+  "Dining Room",
+  "Conservatory",
+  "Kitchen",
+  "Study",
+  "Library",
+  "Billiard Room",
+  "Lounge",
+  "Ballroom",
+  "Hall",
+  "Spa",
+  "Living Room",
+  "Observatory",
+  "Theater",
+  "Guest House",
+  "Patio"
+];
 
-const weapons = []
+const months = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
```

Comment:
What is this? :)

Reply:
This is my months-list where each number represents the number of days a month. For e.g. months[0] returns 31 which is the number of days in January. With the help of the pre-made randomSelector function it returned a number value. As I wrote in the readme I didn't take non leap years in to account that's why months[1] is 29 insted of 28. I used it both to randomise month and day in the month. 

Not sure if that explanation made sense, but I added the code below and bolded how I used it! Basically I needed the months list to pick both month and also day. This is most def not the prettiest solution but I was fun to make something all by myself.

`const pickDate = () => {
  let year = randomDateSelector(1919, 2019);
  let **month** = randomSelector(**months**);
  let day = randomDateSelector(1, **month**);
  if (month === 29) {
    month = "02";
    day = addZero(day);
  } else if (month % 2 === 0) {
    month = randomSelector(monthsEven);
    day = addZero(day);
  } else {
    month = randomSelector(monthsOdd);
    day = addZero(day);
  }`

Reply:
At least I thought I bolded the important data types 🙈

Reply:
Ohhhh, right! I didn't understand that. I though this was the month numbers, you now like month 01 is januari and 12 is december. That is why I was confused. Maybe this variable could be named `daysInMonths` or something that describes it? :)

Reply:
Haha yes that would def be a better name! It felt super obvious for me but then again I was the one that came up with the idea so it's not really a surprise that I understood what the purpose of it! I'll make sure to think twice about variable names in the future!